src/v2/Apps/SavedSearchAlerts/savedSearchAlertsRoutes.tsx

```diff
@@ -0,0 +1,32 @@
+import loadable from "@loadable/component"
+import { AppRouteConfig } from "v2/System/Router/Route"
+import { graphql } from "relay-runtime"
+
+const SavedSearchAlertsApp = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "savedSearchAlertsAppBundle" */ "./SavedSearchAlertsApp"
+    ),
+  {
+    resolveComponent: component =>
+      component.SavedSearchAlertsAppPaginationContainer,
+  }
+)
+
+export const savedSearchAlertsRoutes: AppRouteConfig[] = [
+  {
+    path: "/alerts",
```

Comment:
If this is a user settings kinda thing, similar to the [comment here](https://github.com/artsy/force/pull/9260#discussion_r792937348) related to alerts management, rather than spinning up a whole new app we should locate files under `v2/Apps/Settings/Routes/SavedSearch`, and then just mount this path in `settingsRoutes`, towards the bottom. (We've been trying to contain some of the sprawl of new apps.)

Reply:
These are old changes. I'm just doing a rebase master branch right now in this branch